### PR TITLE
[QNN EP/Quantization] Add MinimumRealRange extra option to quantization script

### PR DESCRIPTION
**Pull request by adrianlizarraga**
### Description
Adds the extra option `MinimumRealRange` to the quantization script:

```python3
"""
MinimumRealRange= float|None :
                    Default is None. If set to a floating-point value, the calculation of the quantization parameters
                    (i.e., scale and zero point) will enforce a minimum range between rmin and rmax. If (rmax - rmin)
                    is less than the specified minimum range, rmax will be set to rmin + QuantMinRealRange. This is
                    necessary for EPs like QNN that require a minimum floating-point range when determining
                    quantization parameters.
"""
```

### Motivation and Context
QNN requires a minimum floating-point range of 0.0001.

### CODE REVIEW

File: onnxruntime/python/tools/quantization/quantize.py [hunk 1]
```diff
@@ -370,6 +370,12 @@ def quantize_static(
                     `com.microsoft` domain, which forces use of ONNX Runtime's QuantizeLinear and DequantizeLinear
                     contrib op implementations. The contrib op implementations may support features not standardized
                     into the ONNX specification (e.g., 16-bit quantization types).
+                QuantMinRealRange = float|None :
```
Comment (adrianlizarraga) [inline onnxruntime/python/tools/quantization/quantize.py:373]: Any suggestions for a more descriptive name?

Comment (yufenglee) [reply onnxruntime/python/tools/quantization/quantize.py:373]: How about MinimumRealRange?

Comment (adrianlizarraga) [reply onnxruntime/python/tools/quantization/quantize.py]: I made the change, thanks.

**Review by yufenglee — APPROVED** [review]
:shipit:

**Pull request merged**